### DISCUSSION TITLE
Mounting the CephFs using ceph-dokan

### DISCUSSION THREAD

**Discussion by mayuripriya12** [dokan-dev/dokany#1052] [Q&A] [answered]
Hi all,

I have set up my cluster and tried to mount the ceph file system(CephFS) on linux and windows using ceph dokan.
I was able to mount the file system in both the OS. But in windows I can only read the file. I am unable to edit the file in windows while in linux edit works perfectly fine.

While editing the file in windows i get the below error message.

![image](https://user-images.githubusercontent.com/75684402/150334780-d0851287-da23-4aeb-b296-dafa9f0253b6.png)

Has anyone know about this issue. It would be a great help. Thanks in advance.

Best Regards,
Mayuri Priya

**Comment by Liryna** [comment, marked as answer]
Hi,

You should reach out to the ceph team for this.
If they find that it is a Dokan issue they can provide details here.
